48-kori-bustard/technical_specification.md

```diff
@@ -0,0 +1,171 @@
+# Notification Orchestration Service (Kori Bustard)
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+**Attention: Please do not put any confidential content here.**
+
+
+## Scope
+### Outline:
+The goal of this epic is to create a base implementation for a new service that
+publishes Notification events upon consuming events corresponding to specific points
+in user journeys.
+A description of the concept can be found in
+[this ADR](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr007_sourcing_notifications.md).
+The name of the new service is the **Notification Orchestration Service (NOS)**.
+
+### Included/Required:
+- Initial implementation of NOS
+- Notification Service Idempotence
+- Addition of new event schemas to ghga-event-schemas
+- Replace ARS notification events
+
+
+## Notification Summary:
+
+### List of Notification Sources
+
+The following list contains notifications and the intended recipients.
+An asterisk by the Purpose field signifies that it's not clear whether the notification
+is needed.
+
+_**Authentication**_
+
+> It needs to be determined whether the Data Steward here is Central or Local
+
+> **IVA**, or **Independent Verification Address**, refers to some physical or digital
+> address, such as a mailing address or phone number, and is used to issue a code for
+> authentication.
+
+| Recipient    | Purpose                                  | Source Exists | Data Req'd |
+|--------------|------------------------------------------|---------------|------------|
+| Data Steward | LS Login doesn’t match what's registered | No            | User ID    |
+| User         | IVA invalidated                          | No            | User ID    |
+| User         | IVA verification code requested          | No            | User ID    |
+| Data Steward | IVA verification code requested          | No            | User ID    |
```

Comment:
Is this user has requested a code or data steward has requested code?

Just wondering, because below the data steward is notified about user behavior

Reply:
The user requests the code. Then the user gets a confirmation notification, and the data steward gets a notification that the user has requested the code.

Reply:
Then this should fit better:
```suggestion
| Data Steward | IVA verification code requested by user  | No            | User ID    |
```